README.md

```diff
@@ -1,72 +1,72 @@
 
-| Directory        | Content                                                   |
-| ---------------- | --------------------------------------------------------- |
-| Boards           | Driver Validation examples for a set of boards            |
-| DoxyGen          | Source of the documentation                               |
-| Include          | Header files for driver validation components             |
-| Source           | Source files for driver validation components             |
-| Tools            | SockServer and SPI_Server source and executable files     |
-| Utilities        | Utility programs                                          |
+### Supported Driver Interfaces
 
-## Generate CMSIS Pack for Release
+The CMSIS-Driver Validation provides tests for the following CMSIS-Driver interfaces:
 
-To build a complete CMSIS pack for installation the following additional tools are required:
- - **doxygen.exe**    Version: 1.9.2 (Documentation Generator)
- - **mscgen.exe**     Version: 0.20  (Message Sequence Chart Converter)
- - **7z.exe (7-Zip)** Version: 16.02 (File Archiver)
-  
-Using these tools, you can generate on a Windows PC:
- - **CMSIS Software Pack** using the batch file **gen_pack.bat** (located in ./Utilities). This batch file also generates the documentation.
- - **CMSIS Documentation** using the batch file **gen_doc.bat** (located in ./Doxygen). 
-
-## License
-
-Arm CMSIS-Driver Validation is licensed under Apache-2.0.
-
-## Contributions and Pull Requests
-Contributions are accepted under Apache-2.0. Only submit contributions where you have authored all of the code.
+- **CAN** - Controller Area Network interface driver.
+- **Ethernet** - Ethernet MAC and PHY peripheral interface driver.
+- **I2C** - Inter-Integrated Circuit multi-master serial single-ended bus interface - driver.
+- **MCI** - Memory Card Interface driver for SD/MMC memory.
+- **SPI** - Serial Peripheral Interface driver.
+- **USART** - Universal Synchronous and Asynchronous Receiver/Transmitter interface driver.
+- **USB Device** - Universal Serial Bus Device interface driver.
+- **USB Host** - Universal Serial Bus Host interface driver.
+- **WiFi** - Wireless Fidelity Interface module/shield driver.
 
-### Issues and Labels
+## Repository Structure
 
-Please feel free to raise an [issue on GitHub](https://github.com/ARM-software/CMSIS-Driver_Validation/issues)
-to report misbehavior (i.e. bugs) or start discussions about enhancements. This
-is your best way to interact directly with the maintenance team and the community.
-We encourage you to append implementation suggestions as this helps to decrease the
-workload of the very limited maintenance team. 
+| Directory/File        | Content                                                   |
+| --------------------- | --------------------------------------------------------- |
+| `./Boards/`           | Driver Validation examples for various boards             |
```

Comment:
I would remove "./" and "/" and, for example, instead of './Boards/' it would be just 'Boards' because if you look at folders via web on the GitHub there are no "./" or "/"

Reply:
In the web view there're different icons for folders and files that make it more clear. But I'll remove the "/" here and make it as links to corresponding items.